src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/LicenceType.kt

```diff
@@ -14,5 +14,11 @@ enum class LicenceType {
       nomisRecord.topupSupervisionExpiryDate == null || nomisRecord.topupSupervisionExpiryDate <= nomisRecord.licenceExpiryDate -> AP
       else -> AP_PSS
     }
+
+    fun getHdcLicenceType(nomisRecord: PrisonerSearchPrisoner) = when {
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of. -->
**note:** ‏Need to double check this with Shirley - I think based on our conversation this is right but also might be better to use the existing `getLicenceType` rule if that also makes sense.

Reply:
**suggestion:** think it probably just makes sense to use `getLicenceType`

You can have PSS HDC licences it just happens very very rarely